src/main/java/com/amazon/dlic/auth/http/jwt/AbstractHTTPJwtAuthenticator.java

```diff
@@ -46,17 +48,20 @@
     private final static Logger log = LogManager.getLogger(AbstractHTTPJwtAuthenticator.class);
 
     private static final String BEARER = "bearer ";
+    private static final Pattern BASIC = Pattern.compile("^\\s*Basic\\s.*", Pattern.CASE_INSENSITIVE);
```

Comment:
According to the specifications the token will always start with "Basic ". I don't think we should allow spaces before.
https://tools.ietf.org/html/rfc2617#page-5

Reply:
As we don't need to ensure that it is a valid "Basic" authentication scheme here, but only need to filter out invalid JWT tokens to avoid JWT validation warnings, it is OK to allow spaces before "Basic" to be consistent with `trim()` used in [HttpHelper.java](https://github.com/opendistro-for-elasticsearch/security/blob/6fe4e0ffca29af61afa8492d7afd5846a3e6753a/src/main/java/com/amazon/opendistroforelasticsearch/security/support/HTTPHelper.java#L48)